app/server/utils/transaction_limits.py

```diff
@@ -11,11 +12,11 @@ def check_user_liquid_token_type(credit_transfer):
     return False
 
 
-def check_group_user_liquid_token_type(credit_transfer):
+def check_group_user_liquid_token_type(credit_transfer: CreditTransfer):
     t = credit_transfer.token
-    if t is not None and t.token_type == token.TokenType.LIQUID:
-        # TODO: add group account check
-        raise NotImplementedError
+    if t is not None and t.token_type == token.TokenType.LIQUID and \
+            credit_transfer.sender_user.has_group_account_role():
```

Comment:
should be 
`credit_transfer.sender_user.has_group_account_role` as `has_group_account_role` is a hybrid property rather than a pure method